master/getting-started/docker/index.md

```diff
@@ -32,15 +32,20 @@ you need to install and use Calico.
 
 ## Tutorials
 
-#### [Simple Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/simple-policy)
+#### [Simple Profile Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/simple-profile-policy)
 
 The above guide demonstrates Calico connectivity cross host, and how to limit
 that connectivity using simple Calico policy.
 
-#### [Advanced Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/advanced-policy)
+#### [Advanced Profile Policy]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/advanced-profile-policy)
 
 The above guide digs deeper into advanced policy configurations for workloads.
```

Comment:
What did you think about the titles Dan and I were discussing
* Security using Calico Profiles
* Security using Calico Profiles and Policy
* Security using Docker Labels and Calico Policy
I think the detail of what happens with the profiles creating labels can be left as a detail in the tutorial.  Do you think it needs to be in the title itself?